[rospy.client][INFO] 2020-09-10 10:21:37,168: init_node, name[/diagnostic_rplidar_head_topic], pid[13117]
[xmlrpc][INFO] 2020-09-10 10:21:37,169: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-09-10 10:21:37,169: Started XML-RPC server [http://cuboid0009:34149/]
[rospy.init][INFO] 2020-09-10 10:21:37,169: ROS Slave URI: [http://cuboid0009:34149/]
[rospy.impl.masterslave][INFO] 2020-09-10 10:21:37,169: _ready: http://cuboid0009:34149/
[rospy.registration][INFO] 2020-09-10 10:21:37,170: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-09-10 10:21:37,170: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-09-10 10:21:37,270: registered with master
[rospy.rosout][INFO] 2020-09-10 10:21:37,270: initializing /rosout core topic
[rospy.rosout][INFO] 2020-09-10 10:21:37,272: connected to core topic /rosout
[rospy.simtime][INFO] 2020-09-10 10:21:37,273: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-09-10 10:21:37,285: topic[/head_scan] adding connection to [http://cuboid0009:46327/], count 0
[rospy.internal][INFO] 2020-09-10 10:21:37,398: topic[/diagnostics] adding connection to [/diag_agg], count 0
[rospy.internal][INFO] 2020-09-10 10:21:37,399: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-09-10 10:23:24,315: topic[/diagnostics] removing connection to /diag_agg
[rospy.impl.masterslave][INFO] 2020-09-10 10:23:25,695: external shutdown request from [/rosnode]: user request
[rospy.core][INFO] 2020-09-10 10:23:25,695: signal_shutdown [external shutdown request from [/rosnode]: [user request]]
[rospy.internal][INFO] 2020-09-10 10:23:25,698: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-09-10 10:23:25,699: topic[/head_scan] removing connection to http://cuboid0009:46327/
[rospy.internal][WARNING] 2020-09-10 10:23:25,699: Unknown error initiating TCP/IP socket to cuboid0009:59671 (http://cuboid0009:46327/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2020-09-10 10:23:25,699: Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 838, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.core][INFO] 2020-09-10 10:23:25,872: signal_shutdown [atexit]
